docs/examples/cart-join.md

```diff
@@ -0,0 +1,256 @@
+In this example, we're going to build a small online order fulfillment
+system. It will join two event streams: one containing customer orders
+and another containing successful payments, and then emit which orders
+for each customer have been paid. We're also going to design it to be
+**recoverable** and restartable after invalid data.
+
+## Sample Data
+
+First let's make a local file containig our sample event data in JSON
+format. Save this as `cart-join.jsons`.
```

Comment:
I usually see this as `json`, not `jsons`. Does `jsons` work?

Reply:
I guess it must 🤯

Reply:
It does work, it's just a file name. I do this to remember that the file isn't a single JSON object, but is an object on each line. But I don't think that's a common convention. It's raising more questions than answers so I'll change this back.

Reply:
makes sense